src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -359,6 +363,7 @@
           Arguments='"@$(_EmccDefaultFlagsRsp)" $(_EmccDefaultFlags) "@$(_EmccCompileRsp)"'
           EnvironmentVariables="@(EmscriptenEnvVars)"
           DisableParallelCompile="$(DisableParallelEmccCompile)"
+          MessageToIndicateCompiling="Compiling native assets with emcc with $(EmccCompileOptimizationFlag). This may take a while ..."
```

Comment:
Could we also have a loud message to Blazor users who don't have workload installed and are running Release/publish

"Publishing without optimizations. Although it's optional for Blazor, we strongly recommend using `wasm-tools` workload! You can install it by running `dotnet workload install wasm-tools` from the command line."

something like that.

Reply:
I'll add this in a follow up PR.